lib/webview-helpers.js

```diff
@@ -292,23 +298,11 @@ helpers.procFromWebview = async function procFromWebview (adb, webview) {
 /**
  * Retrieves webview names for getContexts
  *
- * @param {object} adb - an ADB instance
- * @param {GetWebviewOpts} opts See note on getWebViewsMapping
- *
+ * @param {Array<WebviewsMapping>} webviewsMapping See note on getWebViewsMapping
+ * @param {boolean} ensureWebviewsHavePages Whether to check for webview pages presence
  * @return {Array.<string>} - a list of webview names
  */
-helpers.getWebviews = async function getWebviews (adb, {
-  androidDeviceSocket = null,
-  ensureWebviewsHavePages = true,
-  webviewDevtoolsPort = null,
-  enableWebviewDetailsCollection = null,
-} = {}) {
-  const webviewsMapping = await this.getWebViewsMapping(adb, {
-    androidDeviceSocket,
-    ensureWebviewsHavePages,
-    webviewDevtoolsPort,
-    enableWebviewDetailsCollection
-  });
+helpers.getWebviews = function getWebviews (webviewsMapping, ensureWebviewsHavePages = true) {
```

Comment:
it still makes sense to keep the old signature, since it is easier to add new options to an object rather just as a separate arg

Reply:
Tried to address as https://github.com/appium/appium-android-driver/pull/699/commits/3ebccd4a243e67239d4547a854b10709c37eadae

Let me know if it is something different from your expectation.